app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -77,6 +79,8 @@
     @Inject OkHttpJsonApiClient okHttpJsonApiClient;
     @Inject CampaignsPresenter presenter;
     @Inject LocationServiceManager locationManager;
+    @Inject
```

Comment:
why was this added?

Reply:
Yes, I should remove it, the presenter is doing the job